src/suvela/PracticeArray.java

```diff
@@ -0,0 +1,60 @@
+package suvela;
+
+public class PracticeArray {
+	int min(int[] array) {
+		int index=0;
+		int min = array[index];
+		for (index = 0; index < array.length; index++) {
```

Comment:
video sent for this code on skype, pls check & here you can write code like below as well why you define index separate n then using into for loop.

for(int index = 0;index < arr.length; index++)